packages/api-explorer/src/components/common/index.ts

```diff
@@ -23,5 +23,5 @@
  SOFTWARE.
 
  */
-export { RunItHeading, ApixSection } from './common'
-export { Loader } from './Loader'
+export * from './common'
+export * from './Loader'
```

Comment:
This is fine but I think the common directory kind of sucks. I think the two components that remain should be promoted to components.

Not part of this PR though.

Reply:
Thanks. I may get to that for this PR. If not, we should have a refactor PR soon.